src/js/utils/behavior.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+const assign = require('object-assign');
```

Comment:
Consider using `import` in place of all `require`s. While it wouldn't really have any immediate effect, it would pave the way for tree-shaking in your bundler (either webpack2 or rollup).